test/helper.rb

```diff
@@ -7,12 +8,30 @@
 
 require 'gem2rpm'
 
+# If you want to test in off line environment, set environment variable.
+def local_test?
```

Comment:
Isn't there chance to rewrite this into method something like "skip_offline", so you could later avoid the conditions?